README.md

```diff
@@ -183,6 +183,8 @@ https://github.com/intel/intel-extension-for-transformers/assets/88082706/9d9bdb
 ## 📃Selected Publications/Events
 * NeurIPS'2023 on Efficient Natural Language and Speech Processing: [Efficient LLM Inference on CPUs](https://arxiv.org/abs/2311.00502) (Nov 2023)
 * NeurIPS'2023 on Diffusion Models: [Effective Quantization for Diffusion Models on CPUs](https://arxiv.org/pdf/2311.16133.pdf) (Nov 2023)
+* Blog published on datalearner: [Analysis of the top ten popular open source LLM by HuggingFace in the fourth week of November 2023 - the explosion of multi-modal large models and small-scale models](https://www.datalearner.com/blog/1051701014024122) (Nov 2023)
+* Blog published on zaker: [LLMs inference performance is accelerated up to 40 times using this toolkit](https://app.myzaker.com/news/article.php?pk=656857148e9f0961d70ac3d3) (Nov 2023)
```

Comment:
With this toolkit, the inference performance of large models can be accelerated by 40 times.